hypha/templates/base.html

```diff
@@ -123,6 +123,45 @@
         </script>
         <!-- htmx end -->
 
+        <!-- choices select -->
+        <script type="module">
+            import Choices from "{% static 'js/esm/choices.js-10-2-0.js' %}";
+
+            htmx.onLoad(function() {
+                const selectElements = document.querySelectorAll("[data-js-choices]");
+
+                selectElements.forEach((selectElement) => {
+                    if (!selectElement.choicesInstance) {
+                        const choicesInstance = new Choices(selectElement, {
+                            shouldSort: false,
+                            allowHTML: true,
+                            removeItemButton: true,
+                        });
+                        selectElement.choicesInstance = choicesInstance;
```

Comment:
Not enough to set `selectElement.choicesInstance = true;`?

Reply:
Cleaner solution than my hack in any case.

Reply:
Yes, true should work. It will be more efficient.